blocks/init/src/blocks/custom/columns/columns-editor.scss

```diff
@@ -0,0 +1,13 @@
+// // stylelint-disable declaration-no-important
+
+@import './styles/shared';
+
+body .eightshift-block.block-columns {
```

Comment:
Why is `body` here?

Reply:
Because I want to show WordPress goods who is the stronger one. :D

All jokes aside. This is because of the multiple div nesting inside editor. And I needed to override the selector of 5 index.